openpgp.d.ts

```diff
@@ -336,56 +336,76 @@ declare abstract class BaseSecretKeyPacket extends BasePublicKeyPacket {
 }
 
 export class SecretKeyPacket extends BaseSecretKeyPacket {
-  public tag: enums.packet.secretKey;
+  static readonly tag: enums.packet.secretKey;
 }
 
 export class SecretSubkeyPacket extends BaseSecretKeyPacket {
-  public tag: enums.packet.secretSubkey;
+  static readonly tag: enums.packet.secretSubkey;
```

Comment:
We might want to add a private dummy field `isSubkey()` to the key packets, to prevent passing a subkey when expecting a primary key, and vice versa -- the current classes have compatible structure and TS can't detect the difference.